packages/fxa-auth-server/lib/senders/emails/templates/subscriptionDowngrade/index.mjml

```diff
@@ -7,15 +7,15 @@
 <mj-section>
   <mj-column>
     <mj-text css-class="text-body">
-      <span data-l10n-id="subscriptionDowngrade-content-switch" data-l10n-args="<%= JSON.stringify({productNameNew, productNameOld}) %>">You have successfully switched from <%- productNameOld %> to <%- productNameNew %>.</span>
+      <span data-l10n-id="subscriptionDowngrade-content-switch" data-l10n-args="<%= JSON.stringify({productName, productNameOld}) %>">You have successfully switched from <%- productNameOld %> to <%- productName %>.</span>
```

Comment:
I noticed that Mailer.prototype.supbscriptionDownGrade provdides productNameNew, not productName old.